src/main/java/parser/FormUrlEncodedParsingStrategy.java

```diff
@@ -0,0 +1,29 @@
+package parser;
+
+import static constant.Constant.*;
+
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import http.HttpRequestBody;
+
+public class FormUrlEncodedParsingStrategy implements HttpBodyParsingStrategy {
+	private static final String PARAMETER_EQUAL_SIGN = "=";
+
+	@Override
+	public HttpRequestBody parse(String body) {
+		if (body.isBlank()) {
+			return new HttpRequestBody();
+		}
+
+		try {
+			return new HttpRequestBody(Arrays.stream(body.split(PARAMETER_SEPARATOR))
+				.map(s -> s.split(PARAMETER_EQUAL_SIGN))
+				.collect(Collectors.toMap(arr -> URLDecoder.decode(arr[0], StandardCharsets.UTF_8), arr -> URLDecoder.decode(arr[1], StandardCharsets.UTF_8))));
```

Comment:
아래와 같이 생성자 파라미터에 통째로 들어간 stream 구문을 변수로 추출해주면 조금 더 가독성이 좋아질 것 같아요. 🙂
```suggestion
                         Map<String, String> body = Arrays.stream(body.split(PARAMETER_SEPARATOR))
				.map(s -> s.split(PARAMETER_EQUAL_SIGN))
				.collect(Collectors.toMap(arr -> URLDecoder.decode(arr[0], StandardCharsets.UTF_8), arr -> URLDecoder.decode(arr[1], StandardCharsets.UTF_8)));
			return new HttpRequestBody(body);
```